docs/modules/ROOT/pages/references/architecture/control-api-org.adoc

```diff
@@ -0,0 +1,83 @@
+= {controlapi}: Organization
+
+TIP: This resource implements the xref:references/functional-requirements/portal.adoc#_feature_manage_organizations["Manage Organizations"] features.
+
+List operations on Kubernetes resources can not be filtered by RBAC, it's a binary operation:
+Resources can either be listed or not, there is no way to give access only to a sub-set.
+To circumvent this limitation, `Organization` is a virtual resource.
+
+The `Organization` resource represents a filtered and formatted list of standard Kubernetes `Namespace` resources which have specific labels and annotations.
+
+It is assumed that the resource `appuio.io/v1/Organization` is used for all operations.
+
+== Object
+
+.Virtual resource
+[source,yaml]
+----
+apiVersion: appuio.io/v1
+kind: Organization
+metadata:
+  name: acme-corp <1>
+spec:
+  displayName: Acme Corp. <2>
+----
+<1> `metadata.labels[appuio.io/metadata-name]` from the represented `Namespace` resource
+<2> Replicates `metadata.annotations[organization.appuio.io/display-name]` from the `Namespace`
+
+.Original resource
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: org-acme-corp <1>
+  labels:
+    appuio.io/resource-type: organization <2>
+    appuio.io/metadata-name: amce-corp <3>
+  annotations:
+    organization.appuio.io/display-name: Acme Corp. <4>
+----
+<1> Prefixed (`org`) resource name to circumvent possible name collision (e.g. if a company is called "kube-system")
+<2> Identify resource type, used by the API server to filter for namespace representing organizations
+<3> Resource name of the virtual `Organization` object
+<4> Reflected in the organization object as `spec.displayName`
+
+== Labels and Annotations
+
+[cols="2,1,1,3",options="header"]
+|===
+|Name
+|Type
+|Resource
+|Description
+
+|`appuio.io/resource-type`
+|label
+|`v1/Namespace`
+|Identifies the resource type in the scope of the {controlapi}
+
+|`organization.appuio.io/display-name`
+|annotation
+|`appuio.io/v1/Organization`
+|Display name of the organization
+
+|===
+
+== Resource filter
+
+The virtual resource is a filtered view on `Namespaces`.
+The filter uses the following heuristic:
+
+* Resource kind = v1/Namespace
+* label[appuio.io/resource-type] = organization
+* subject is bound to one of the defined `ClusterRole` resources.
+
+== RBAC rules
+
+These are `ClusterRole` resources which are bound to a subject by a namespaced `RoleBinding`:
+
+`org-view`:: View (read only) access to an organization
+`org-admin`:: Admin (read / write) access to an organization
+
+Creating organizations can be done by all authenticated users.
```

Comment:
There might be cases where this is undesirable for certain resellers.
For example, an IdP brokering might "lock" the creation of organizations and users are locked in certain organizations defined by such an IdP.
However, such locking should be easy to achieve via configurable RBAC rules.